MVPproject/MVPproject/RecipesModule/Cell/TypeRecipesCollectionViewCell.swift

```diff
@@ -0,0 +1,98 @@
+// TypeRecipesCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячека типа рецепта
+class TypeRecipesCollectionViewCell: UICollectionViewCell {
+    // MARK: - Constants
+
+    static let reuseID = "TypeRecipesCollectionViewCell"
+
+    // MARK: - Visual Components
+
+    private let titleImageView = UIImageView()
+
+    private let bottomView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGray
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
это не обязательно использовать, но я создала экстеншн для более удобного отключения маски. Ты можешь использовать тот способ, который тебе удобнее
```suggestion
        view.disableAutoresizingMask()
```